libs/pages/application/src/lib/slices/environment-variable-ui.slice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { RootState } from '@console/store/data'
+
+export const ENVIRONMENT_VARIABLE_UI_FEATURE_KEY = 'environmentVariable'
```

Comment:
I think we don't use a custom slice for this, we can have use context for specific store on page

To me, slices should be used for global store only